docs/home/changelog/hotfixes-2-16.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Changelog Version 2.16
+summary: This article lists the fixed and known issues in the patch versions as well as additional new features that were added in each patch version.
+authors:
+    - Jason Novich
+date: 2024-Feb-26
+---
+
+The following is a list of the known and fixed issues for Run:ai V2.16.
+
+## Version 2.16.21
+
+| Internal ID | Description |
+|--|--|
+| RUN-16463 | Fixed an issue after a cluster upgrade to v2.16, where some metrics of pre-existing workloads were displayed incorrectly in the *Overview* Dashboard. |
+
+## Version 2.16.18
+
+| Internal ID | Description |
+|--|--|
+| RUN-16486 | Fixed an issue in the *Workloads* creation form where the GPU fields of the compute resource tiles were showing no data. |
+
+## Version 2.16.16
+
+| Internal ID | Description |
+|--|--|
+| RUN-16340 | Fixed an issue in the *Workloads* table where filters were not saved correctly. |
+
+## Version 2.16.15
+
+### Release content
+
+* <!-- RUN-12664 - [Control-plane] Implement get workload/pods API -->Implemented a new Workloads API to support the *Workloads* feature.
+
+### Fixed issues
+
+| Internal ID | Description |
+|--|--|
+| RUN-16346 | Fixed an issue where the *Jobs* and *Deployments* pages were not displayed for trial users. |
+| RUN-16070 | Fixed an issue where missing metrics caused the *Nodepools* table to appear empty. |
+
+## Version 2.16.14
+
+### Release content
+
+*<!-- RUN-16108 | Reduce the frequency of fetching metrics from 10 sec to 30 sec -->Improved overall performance by slowing down metrics updates from 10 seconds to 30 seconds.
+
+### Fixed issues
+
+| Internal ID | Description |
+|--|--|
+| RUN-16255 | Fixed an issue in the *Analytics* dashboard where some node panels were displaying incorrect data. |
```

Comment:
It's not the nodepools that showed incorrect data. Look at the bug - Natasha listed the specific 2 widgets. Nodepools are the cause.
This one needs to be rewritten.